packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -330,6 +335,7 @@ const SceneView = ({
       hasLargeHeader={options.headerLargeTitle ?? false}
       customAnimationOnSwipe={animationMatchesGesture}
       fullScreenSwipeEnabled={fullScreenGestureEnabled}
+      fullScreenSwipeShadowEnabled={fullScreenGestureShadowEnabled}
```

Comment:
Remember, that you need to ignore this file with `@ts-expect-error`, since this prop is not available yet 😄

```suggestion
      // @ts-expect-error prop supported from react-native-screens 3.33.0 onwards
      fullScreenSwipeShadowEnabled={fullScreenGestureShadowEnabled}
```